provisioner/README.md

```diff
@@ -94,6 +94,20 @@ ec2_xtra:
     username: ec2-user
     os_type: linux
     size: r5b.2xlarge
+
+# Registry name to download execution environments
+ee_registry_name: registry.redhat.io
+
+# List of execution environments to download during controller installation:
+ee_images:
+   - "{{ ee_registry_name }}/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.1"
+   - "{{ ee_registry_name }}/ansible-automation-platform-20-early-access/ee-29-rhel8:2.0.1"
+   - "{{ ee_registry_name }}/ansible-automation-platform-20-early-access/ee-minimal-rhel8:2.0.1"
```

Comment:
AAP 2.1 controller installation should auto install the proper images. Also these images are early access not latest GA.

Reply:
@rlopez133 . Agreed it's the incorrect version. What about the scenario where the new versions of the EEs are released asynchronous to AAP version. I believe having these vars will accommodate that

Reply:
@craig-br true, they likely just need to be optional.

Reply:
@rlopez133 I've updated the EE images to AAP 21. The vars I'm proposing are all optional. The examples are specified under the `Optional` section of the `extra_vars.yml` README